src/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh

```diff
@@ -105,8 +105,14 @@ __vsc_precmd() {
 
 __vsc_preexec() {
 	__vsc_initialized=1
-	if [[ ! "$BASH_COMMAND" =~ ^__vsc_prompt* ]]; then
-		__vsc_current_command=$BASH_COMMAND
+	if [[ -z $BASH_COMMAND ]]; then
+		__vsc_command= "$(which $BASH_COMMAND)"
+		__vsc_command= basename __vsc_command
```

Comment:
BASH_COMMAND should have the full command line so we wouldn't be able to run `basename` on it.

Reply:
yeah currently working on a different idea 
"The first word of each simple command, if unquoted, is checked to see if it has an alias. If so, that word is replaced by the text of the alias. " so maybe we can quote it

Reply:
already tried unsetting the expand alias option at the top of the file and setting it at the bottom, but that didn't work - maybe permissions? does work when I manually unset that in the terminal though

Reply:
but also "Aliases are expanded when a command is read, not when it is executed." so idk if that's even possible

Reply:
I don't think this is doable atm without storing all aliases and checking each (slow) since the moment the command is read, it's expanded